tests/subresource_api_test.go

```diff
@@ -237,6 +241,164 @@ var _ = Describe("[sig-compute]Subresource Api", func() {
 				}, 90*time.Second, 1*time.Second).Should(Equal(v1.Running))
 			})
 		})
+
+		Context("ExpandInstancetype endpoint", func() {
+			Context("instancetype", func() {
+				var (
+					instancetype *instancetypev1alpha1.VirtualMachineInstancetype
+					expectedCpu  *v1.CPU
+					vmi          *v1.VirtualMachineInstance
+				)
+
+				BeforeEach(func() {
+					instancetype = newVirtualMachineInstancetype(nil)
+					instancetype.Spec.CPU.Guest = 2
+
+					expectedCpu = &v1.CPU{
+						Sockets: 2,
+						Cores:   1,
+						Threads: 1,
+					}
+
+					var err error
+					instancetype, err = virtCli.VirtualMachineInstancetype(util.NamespaceTestDefault).
+						Create(context.Background(), instancetype, metav1.CreateOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					vmi = libvmi.NewCirros()
+					// Removing memory requests, because they are set in instancetype
```

Comment:
FWIW there's some code in the instancetype func tests we could share and reuse here:

https://github.com/kubevirt/kubevirt/blob/74781324b726a6d58799be1257a34f61e6749513/tests/instancetype_test.go#L627-L639

Reply:
This is not needed, because `libvmi.New()` does not fill resources or devices.

Reply:
ACK thanks I'll clean up the existing tests in that case.